_config.yml

```diff
@@ -1,5 +1,4 @@
 name: Before you ship
-baseurl: /before-you-ship
```

Comment:
Oh, in that case, I can add it back, but if all links are prefixed with `{{ site.baseurl }}`, everything should still work.